src/services/item/ws/hooks.ts

```diff
@@ -125,9 +127,12 @@ function registerMemberItemsTopic(websockets: WebsocketService, itemService: Ite
   // - notify own items of creator of deleted item IF path is root
   // - notify members that have memberships on this item of delete
   //   (before with prehook, otherwise memberships lost on cascade from db!)
+  // todo: currently items can be hard deleted without going to the trash
+  // --> preventing this behavior leads to delete the following code
```

Comment:
I don't think we necessarily need to remove the posthook even with the recycle bin? E.g. it should still be true that if an item is recycled then permanently deleted, the change should be broadcasted?